test-requirements.txt

```diff
@@ -1,3 +1,11 @@
-flake8==3.3.0
+# General
 tox
+
+# Docs
 Sphinx
+
+# Testing
+flake8==3.3.0
+nose==1.3.7
+testtools==2.3.0
+codecov
```

Comment:
we will be using it, but since the coverage is so low today I decided to turn it off for now.